libraries/SuplaDevice/examples/Pzem_V_2/Pzem_V_2.ino

```diff
@@ -0,0 +1,78 @@
+/*
+   Copyright (C) AC SOFTWARE SP. Z O.O.
+
+   This program is free software; you can redistribute it and/or
+   modify it under the terms of the GNU General Public License
+   as published by the Free Software Foundation; either version 2
+   of the License, or (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU General Public License for more details.
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+   */
+
+#include <SPI.h>
+#include <SuplaDevice.h>
+#include <io.h>
+#include <supla/sensor/PzemV2.h>
+
+// Choose proper network interface for your card:
+// Arduino Mega with EthernetShield W5100:
+#include <supla/network/ethernet_shield.h>
+// Ethernet MAC address
+uint8_t mac[6] = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05};
+Supla::EthernetShield ethernet(mac);
```

Comment:
remove ethernet shield variant from this example. Currently electricity meter channel does not work with arduino mega board.